debian/libmiral7.symbols

```diff
@@ -67,6 +67,13 @@ libmiral.so.7 libmiral7 #MINVER#
  (c++)"miral::ExternalClientLauncher::snapcraft_launch(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const@MIRAL_5.0" 5.0.0
  (c++)"miral::ExternalClientLauncher::split_command(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)@MIRAL_5.0" 5.0.0
  (c++)"miral::ExternalClientLauncher::~ExternalClientLauncher()@MIRAL_5.0" 5.0.0
+ (c++)"miral::IdleListener::IdleListener()@MIRAL_5.0" 5.0.0
+ (c++)"miral::IdleListener::IdleListener(miral::IdleListenerParams const&)@MIRAL_5.0" 5.0.0
+ (c++)"miral::IdleListener::IdleListener(std::function<void ()> const&, std::function<void ()> const&)@MIRAL_5.0" 5.0.0
+ (c++)"miral::IdleListener::on_dim(std::function<void ()> const&)@MIRAL_5.0" 5.0.0
+ (c++)"miral::IdleListener::on_off(std::function<void ()> const&)@MIRAL_5.0" 5.0.0
+ (c++)"miral::IdleListener::on_wake(std::function<void ()> const&)@MIRAL_5.0" 5.0.0
+ (c++)"miral::IdleListener::operator()(mir::Server&) const@MIRAL_5.0" 5.0.0
```

Comment:
Miral 5.0 was Mir 2.17, These need to go in 5.1 (Except the ones that don't exist)